core/src/main/java/novoda/lib/sqliteprovider/util/DBUtils.java

```diff
@@ -125,31 +131,38 @@ public static String getSQLiteVersion() {
     @Deprecated
     public static List<String> getUniqueConstrains(SQLiteDatabase db, String table) {
         List<String> constrains = new ArrayList<String>();
```

Comment:
Yeah, this one is "correct" i.e. shouldn't be changed. It's a deprecated method and it used to refer to "constrains". Yeah... even though that should never have been a thing. Anyway, the new stuff uses "constraints", and the old stuff will die.